{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py

```diff
@@ -101,18 +101,27 @@
 
 TEMPLATES = [
     {
-        "BACKEND": "django_jinja.backend.Jinja2",
-        "APP_DIRS": True,
-        "OPTIONS": {
-            "match_regex": r"^(?!admin/.*)",
-            "match_extension": '.html',
-            "newstyle_gettext": True,
+        'BACKEND': 'django_jinja.backend.Jinja2',
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'match_regex': r'^(?!!(admin|registration)/.*)',
+            'match_extension': '.html',
+            'newstyle_gettext': True,
+            'context_processors': [
+                '{{ cookiecutter.project_name }}.base.helpers.settings',
+            ],
+            'filters': {
```

Comment:
Thoughts on using decorators for registering these instead? http://niwinz.github.io/django-jinja/#_registring_filters_in_a_django_way